InnerEye/ML/configs/classification/DummyMulticlassClassification.py

```diff
@@ -0,0 +1,51 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+from typing import Any
+
+import pandas as pd
+
+from InnerEye.ML.scalar_config import ScalarLoss, ScalarModelBase
+from InnerEye.ML.utils.split_dataset import DatasetSplits
+from InnerEye.Common.fixed_paths_for_tests import full_ml_test_data_path
+
+
+class DummyMulticlassClassification(ScalarModelBase):
+    "A config file for dummy image classification model for debugging purposes"
+
+    def __init__(self) -> None:
+        num_epochs = 4
+        super().__init__(
+            local_dataset=full_ml_test_data_path("classification_data_multiclass"),
+            image_channels=["blue"],
+            image_file_column="path",
+            label_channels=["blue"],
+            class_names=["class0", "class1", "class2", "class3", "class4"],
+            # labels_exclusive=False,
```

Comment:
is there test coverage for the two cases labels_exclusive=True and =False?

Reply:
I see no other mention of that in the PR, so I presume it is dead code. This means that the code is always assuming that this is a multi-label classification task, and will allow multiple labels per sample. I think this should be more carefully controlled, we don't want anyone to accidentally feed in labels that are not mutually exclusive into a model that assumes exclusion.

Reply:
yes this is a good point let me take a look